New blog post: 2021 Recap (#121)
AntennaPod/antennapod.github.io: _blog/2021-12-29-2021 Recap.md

---
title: 2021 Recap
excerpt: What happened in the AntennaPod project in 2021

> date: "2021-12-29 11:00:00"

Reviewer: ```suggestion
date: "2021-12-30 11:00:00"
```

image: "2021/recap.png"
author: bytehamster
layout: blog
---

Looking back at 2021, a lot happened in the AntennaPod community. In this blog post, we want to summarize the most important milestones.

# App development

> **Over the course of 2021**, 48 contributors helped improve AntennaPod. In total, they made no less than 482 changes (commits) to the source code, touching about 29 000 lines of code. This makes AntennaPod a pretty active project, even though it is run completely by volunteers in their free-time. To give everyone access to the app improvements, we released 9 new updates, of which 3 contained major new features.

Reviewer: ```suggestion
**Over the course of 2021**, 48 contributors helped improve AntennaPod. In total, they made no less than 482 changes (commits) to the source code, touching about 29 000 lines of code. This makes AntennaPod a pretty active project, even though it is run completely by volunteers in their free time.

To give everyone access to the app improvements, we released 9 new updates, of which 3 contained major new features.
```
(There currently is no white line, and in retrospect this last sentence better fits with the next bit.)

> For the beginning of 2022, we have already developed a release that is nearly ready. The new release of AntennaPod - version 2.5 - will add support for synchronizing your subscriptions with your own Nextcloud instance. That makes it easier than ever to enjoy sync while still keeping your listening habits private!

Reviewer: ```suggestion
For the beginning of 2022, we have already developed a release that is nearly ready. This new release of AntennaPod -- version 2.5 -- will add support for synchronizing your subscriptions with your own Nextcloud instance. That makes it easier than ever to enjoy sync while still keeping your listening habits private!
```

# Donations

> After a long period of not accepting donations and users asked for it, in **March** this year we decided that we will accept donations. Active community members [discussed](https://forum.antennapod.org/t/should-antennapod-begin-accepting-donations/249/29) what might and should not be paid for with the money, and finally developed [spending guidelines](https://forum.antennapod.org/t/donation-expenses-policy/272/2). In nine months we already received € 1263.75. The first and only spending this year was € 21.60 for a Google Play Console account (to better manage releases). We hope we can put more of the money to good use going forward.

Reviewer: ```suggestion
After a long period of not accepting donations and users asked for it, in **March** this year we decided that we will accept donations. Active community members [discussed](https://forum.antennapod.org/t/should-antennapod-begin-accepting-donations/249/29) what might and should not be paid for with the money, and finally developed [spending guidelines](https://forum.antennapod.org/t/donation-expenses-policy/272/2). In nine months we already received € 1 263.75. The first and only spending this year was € 21.60 for a Google Play Console account (to better manage releases). We hope we can put more of the money to good use going forward.
```

# Hackathon
From **July** 9 to July 11, we organized the first (virtual) AntennaPod Hackathon. After an announcement and public polls on the forum, we decided on a date that fits best with the timezones that our contributors live in. We then held a kick-off meeting where we discussed which parts of AntennaPod we plan to work on next. The kick-off meeting was attended by @ByteHamster, @keunes, @Matth78, @tonytamsf and @ueen, with a short visit by @peakvalleytech. Most specifically, we discussed where we want AntennaPod's user interface to develop in the next years. We agreed on the following long-term changes:
- Developing a [home screen](https://github.com/AntennaPod/AntennaPod/issues/3952) with a quick overview over the app.
- Adding [another episode state "ignored"](https://github.com/AntennaPod/AntennaPod/issues/5237), which you can use if you are not interested in an episode, so you do not have to mark episodes as played that are not actually played.
- Renaming the "new" screen to ["inbox"](https://github.com/AntennaPod/AntennaPod/issues/5267) and adding it directly to the side drawer.

> The Hackathon was very productive, and will explore in which form we can continue such live developer interaction in 2022.

Reviewer: ```suggestion
The Hackathon was very productive, and we will explore in which form we can continue such live contributor interaction in 2022.
```

# Forum
As mentioned above, our forum played an important role in our community. It is the main place where everyone -- from active contributors to occasional app users -- meets, gets support and discusses features. We can already tell that this year well over 350 topics (discussion threads) were created, filled with more than 2 000 posts. Almost 500 users registered to the forum, bringing the total up to 700+ users.
On January 1st, the great forum software (Discourse) [will](https://forum.antennapod.org/t/2021-the-year-in-review) automatically publish a full overview of the forum activity. (So check in on our forum in a few days if you're reading this in 2021.)

# See you again next year
We hope that you had a good year, despite the global pandemic. Having to stay at home is a good time to listen to podcasts, after all 😉 Now, we wish you a good transition into the year 2022, and we will see you again next year!

> PS. Don't hesitate to create a post on our forum if you have any comments about this blog post. As said, it also is a good place to discuss new features and help shape the future of AntennaPod. And if you want to help to make AntennaPod even greater, have a look at our website where you can [contribute](/contribute) in various ways such as translating or coding.

Reviewer: ```suggestion
PS. Don't hesitate to create a post on our forum if you have any comments about this blog post. As said, it also is a good place to discuss new features and help shape the future of AntennaPod. And if you want to help to make AntennaPod even greater, have a look at our website where we introduce the various ways in which you can [contribute](/contribute) to the project, such as translating or coding.
```
